web/packages/teleport/src/TopBar/TopBar.tsx

```diff
@@ -109,6 +109,38 @@ export function TopBar() {
     const oldPrefix = cfg.getClusterRoute(clusterId);
 
     const newPath = history.location.pathname.replace(oldPrefix, newPrefix);
+
+    // TODO (avatus) REMOVE IN 15 (LEGACY RESOURCES SUPPORT)
+    // this is a temporary hack to support leaf clusters _maybe_ not having access
+    // to unified resources yet. When unified resources are loaded in fetchUnifiedResources,
+    // if the response is a 404 (the endpoint doesnt exist), we:
+    // 1. push them to the servers page (old default)
+    // 2. set this variable conditionally render the "legacy" navigation
+    // When we switch clusters (to leaf or root), we remove the item and perform the check again by pushing
+    // to the resource (new default view).
+    window.localStorage.removeItem(KeysEnum.UNIFIED_RESOURCES_DISABLED);
```

Comment:
If we simply remove this item, we get rid of an escape hatch to allow customers to get back to the old view if they encounter a problem with the new one and they want to use the legacy until we implement the fix. Perhaps we could do it differently: instead of simply manipulating the global flag, store a set of clusters for which the unified resources feature is disabled, and combine it with this global flag? I.e. if there's no global flag (or it's set to `'false'`), we show the new UI. If we are on a cluster that we detected not to support the new view (using a key-value relationship in local storage), we don't show the new UI, but only if the global flag is not set. And here, instead of removing the global flag, we simply remove given cluster from the list of clusters for which we keep these disabled.

I think this way we would much the release much safer for us and decrease potential for negative customer experience.

Reply:
I think we can split it into two and get the same effect. `UNIFIED_RESOURCES_NOT_SUPPORTED` and `UNIFIED_RESOURCES_DISABLED`. that way they are both explicit in what they do